microk8s-resources/wrappers/microk8s-istioctl.wrapper

```diff
@@ -11,6 +11,11 @@ if [ ! -f "${SNAP_DATA}/bin/istioctl" ]; then
 fi
 
 source $SNAP/actions/common/utils.sh
+
+if echo "$*" | grep -v -q -- '--kubeconfig'; then
```

Comment:
Should this be abstracted out to a separate function in microk8s-resources/actions/common/utils.sh?

Reply:
Can I please keep this? I like this oneliner better than creating a function. I know it is in a couple of places but it is a singe call to grep.

Reply:
Sure.